uk.co.compucorp.civicrm.tasksassignments/views/dashboard/key-dates.html

```diff
@@ -35,7 +35,7 @@ <h3 class="panel-title">View Dates By:</h3>
                         </div>
                         <div class="form-group">
                             <label for="{{prefix}}filter-date-until" class="col-xs-12 col-sm-3 control-label">Until</label>
-                            <div class="input-group col-xs-12 col-sm-9  col-lg-7">
+                            <div class="input-group col-xs-12 col-sm-9  col-lg-8">
```

Comment:
You can try increasing label to `col-sm-4` to give more breathing space and complete total of "12"